dists/go-ipfs/current

```diff
@@ -1 +1 @@
-v0.12.1
+v0.12.2
```

Comment:
This had to be done manually/carefully to avoid `v0.11.1` being here. Filed for tracking purposes #680